scylla/tests/integration/execution_profiles.rs

```diff
@@ -46,9 +47,13 @@ impl<const NODE: u8> BoundToPredefinedNodePolicy<NODE> {
 }
 
 impl<const NODE: u8> LoadBalancingPolicy for BoundToPredefinedNodePolicy<NODE> {
-    fn pick<'a>(&'a self, _info: &'a RoutingInfo, cluster: &'a ClusterData) -> Option<NodeRef<'a>> {
+    fn pick<'a>(
+        &'a self,
+        _info: &'a RoutingInfo,
+        cluster: &'a ClusterData,
+    ) -> Option<(NodeRef<'a>, Shard)> {
         self.report_node(Report::LoadBalancing);
-        cluster.get_nodes_info().iter().next()
+        cluster.get_nodes_info().iter().next().map(|node| (node, 0)) // FIXME: hardcoded shard 0
```

Comment:
Let's not leave this `TODO` here, but rather discuss it straight and remove.
In the particular case of this test, hardcoded shard 0 seems to work good enough. Another option is to sample a random shard with a fixed seed, but I'm not convinced it's beneficial for the test in any way.

Reply:
This is just a test, so I think using shard 0 is ok. I'll remove the comment.